.github/workflows/media-service-func-ci.yml

```
@@ -1,185 +1,185 @@
-#             az storage blob upload-batch \
-#               --account-name $STORAGE_ACCOUNT_NAME \
-#               --account-key $ACCOUNT_KEY \
-#               --destination $CONTAINER_NAME \
-#               --source ./publish \
-#               --overwrite
-
-#             echo "Restarting Function App to pick up new files..."
-#             az functionapp restart --name $FUNCTION_APP_NAME --resource-group ${{ vars.AZURE_RESOURCE_GROUP }}
-
-#             echo "✅ Deployment Complete"
+name: MediaService Func CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)